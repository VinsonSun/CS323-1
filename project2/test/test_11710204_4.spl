int test_11710204_4(int x, int y)
{
  return x + y;
}
int test_11710204_4(int x, int y)
{
  return x - y;
}
