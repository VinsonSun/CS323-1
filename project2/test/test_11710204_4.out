Error type 4 at Line 5: redefine function: test_11710204_4
